canopen/objectdictionary/__init__.py

```diff
@@ -15,17 +15,22 @@
 logger = logging.getLogger(__name__)
 
 
-def export_od(od, dest: Union[str, TextIO, None] = None, doc_type: Optional[str] = None):
-    """ Export :class: ObjectDictionary to a file.
+def export_od(
+    od: ObjectDictionary,
+    dest: Union[str, TextIO, None] = None,
+    doc_type: Optional[str] = None
+) -> Union[str, None]:
+    """Export an object dictionary.
 
     :param od:
-        :class: ObjectDictionary object to be exported
+        The :class:`!ObjectDictionary` object to be exported.
     :param dest:
-        export destination. filename, or file-like object or None.
-        if None, the document is returned as string
-    :param doc_type: type of document to export.
-       If a filename is given for dest, this default to the file extension.
-       Otherwise, this defaults to "eds"
+        The export destination as a filename, a file-like object, or None.
+        If ``None``, the document is returned as string.
```

Comment:
Further testing reveals that this is not true; we're delegating to the stdlib `configparser` library here, which writes to `sys.stdout` (which _hopefully_ is the actual stdout, and not a monkey-patched thingy). Moreover, this is not tested; I'll add tests in a separate PR.

```suggestion
        If ``None``, the document is printed to :data:`sys.stdout`.
```

Alternatively:

```suggestion
        If ``None``, the document is written to :data:`sys.stdout`.
```

Reply:
I went for the former: see commit bc4f739997d864580ac378dce31afb11418425f2.